Resources/ServerInfo/Rules.txt

```diff
@@ -40,12 +45,13 @@ If you have any questions about these rules, please use the admin help (ahelp) m
 
 [color=#a4885c]4.[/color] Follow Metagaming guidelines
     - If you die and get revived, do not act on things you saw while you were dead.
-    - Do not engage in meta-communications. This includes using chat channels outside of what is available in-game to communicate with other players in the same game. 
+    - Do not engage in meta-communications. This includes using chat channels outside of what is available in-game to communicate with other players in the same game.
     - Players are allowed to have in-character relationships (friends, enemies, or otherwise), however they cannot be used as a reason to grant or deny things based exclusively on having a relationship with one another(i.e. Meta-Friending).
     - You are allowed to have knowledge of past experiences with someone else in prior shifts. This does not give you permission to hold a grudge against someone that results in you treating them differently in an unfair way (i.e. Meta-Grudging).
     - Specific players who were antagonists in previous rounds must not be treated differently because of it.
     - Do not "Antag Roll." This is the act of joining rounds for the purpose of seeing if you joined as an antagonist, and leaving soon after if not. Players who have a history of this behavior will have their whitelist revoked and/or face a ban.
     - [color=#ff0000]Do not stream the current round to the Delta-V Discord.[/color]
+    - Do not place players into cryosleep unless they have given consent to do so, they are fully catatonic, or they have been sentenced to preservative stasis. Always examine a character to double check if they are SSD or catatonic prior to placing them into cryosleep.
```

Comment:
Not to be pedantic but given this will be read by new players this should probably explain the difference

i.e "... if they are SSD (yellow text when you inspect them) or catatonic (purple text when you inspect them)..." 

Or it should contain snippets of the associated texts. I see questions in #help sometimes of people asking the difference between the two or confusing the two.